c-layer/lib.cxx

```diff
@@ -132,16 +138,39 @@ void free_status(struct Status *status) noexcept {
 
 static zypp::Resolvable::Kind kind_to_zypp_kind(RESOLVABLE_KIND kind) {
   switch (kind) {
-    case RESOLVABLE_PACKAGE: return zypp::Resolvable::Kind::package;
-    case RESOLVABLE_SRCPACKAGE: return zypp::Resolvable::Kind::srcpackage;
-    case RESOLVABLE_PATTERN: return zypp::Resolvable::Kind::pattern;
-    case RESOLVABLE_PRODUCT: return zypp::Resolvable::Kind::product;
-    case RESOLVABLE_PATCH: return zypp::Resolvable::Kind::patch;     
+  case RESOLVABLE_PACKAGE:
+    return zypp::Resolvable::Kind::package;
+  case RESOLVABLE_SRCPACKAGE:
+    return zypp::Resolvable::Kind::srcpackage;
+  case RESOLVABLE_PATTERN:
+    return zypp::Resolvable::Kind::pattern;
+  case RESOLVABLE_PRODUCT:
+    return zypp::Resolvable::Kind::product;
+  case RESOLVABLE_PATCH:
+    return zypp::Resolvable::Kind::patch;
   }
   return zypp::Resolvable::Kind::nokind; // should not happen
```

Comment:
```suggestion
  raise "all RESOLVABLE_KIND values should be handled in `switch`";
```

Reply:
As discussed in person, we will not use exceptions and instead create for such cases macro that will print message to stderr and abort.